primitives/src/pcs/multilinear_kzg/util.rs

```diff
@@ -15,6 +15,7 @@ use ark_poly::{
 use ark_std::{end_timer, format, log2, start_timer, string::ToString, sync::Arc, vec, vec::Vec};
 
 /// Evaluate eq polynomial. use the public one later
+#[allow(dead_code)]
```

Comment:
What's the context for it becoming a dead code?

Reply:
it was used in the `gen_srs_for_testing()` only, which is for `#[cfg(any(test, feature = "test-srs"))]`.